index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Henary's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <a href = "index.html" class="brand">My Logo</a>
+        </div>
```

Comment:
You did great making the changes requested by the previous reviewer. I still see one little thing that you need to do to finish this project. 

- [ ] On the Figma Design as shown below, the logo has a blue color without any underline. This underline is the default style of the `a` tag. Kindly remove this underline with `text-decoration: none;`, then change color of the text to the correct color.

Your design
![image](https://user-images.githubusercontent.com/84156714/212470744-5bbca17a-8eb1-4b86-b7cc-b65a410aabd6.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/212470790-4efb8c06-ad92-4af3-a239-c4fbe5ebea4f.png)

Reply:
Thanks for pointing out this mistake. I will fix it now.